.dev/compose.backbone.yml

```diff
@@ -108,7 +108,7 @@ services:
     image: postgres
     environment:
       - PGPASSWORD=Passw0rd
-    command: /bin/bash -c 'env && apt update -y && apt install -y wget && wget https://raw.githubusercontent.com/nmshd/backbone/${BACKBONE_VERSION}/setup-db/setup-postgres.sql -O /setup-postgres.sql && psql -h postgres -U postgres -d enmeshed -f /setup-postgres.sql'
+    command: /bin/bash -c 'env && apt update -y && apt install -y wget && wget https://raw.githubusercontent.com/nmshd/backbone/${BACKBONE_VERSION}/scripts/sql/postgres/setup.sql -O /setup-postgres.sql && psql -h postgres -U postgres -d enmeshed -f /setup-postgres.sql'
```

Comment:
Was this renamed by the Backbone or where does this change come from?

Reply:
yeah, and I didn't want someone else to wonder why the backbone doesn't start